src/libraries/System.Text.Json/src/System/Text/Json/JsonPropertyDictionaryJsonPropertyValueList.cs

```diff
@@ -0,0 +1,206 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Text.Json.Serialization.Metadata;
+
+namespace System.Text.Json
+{
+    internal sealed class JsonPropertyDictionaryJsonPropertyValueList : IList<JsonPropertyInfo>
```

Comment:
That type might warrant a `DebuggerDisplayAttribute`, possible needed in some of the other types like JsonTypeInfo and JsonPropertyInfo.

Reply:
I'll leave this off for now since we don't usually do that for all `IList<T>`